Gemfile

```diff
@@ -93,7 +93,9 @@ gem 'sass-rails', '~> 5.0' # Use SCSS for stylesheets
 gem 'scss_lint', require: false, group: %i[development]
 gem 'secure_headers'
 gem 'selenium-webdriver', '3.142.7', group: %i[test]
-gem 'sentry-raven' # Notch8 Sentry Error Reporting
+gem 'sentry-rails' # Notch8 Sentry Error Reporting
+gem 'sentry-ruby' # Notch8 Sentry Error Reporting
+gem 'sentry-sidekiq' # Notch8 Sentry Error Reporting
```

Comment:
UTK [uses GoodJob](https://github.com/notch8/utk-hyku/blob/a503f83d6ba9c02db680b18a60d0cc0e70592a1f/ops/production-deploy.tmpl.yaml#L124-L125) instead of Sidekiq. Is there a sentry gem for GoodJob? Or any other GoodJob-specific config we should do for the Sentry integration? Either way, I think we can safely remove this gem